vue3-ts-quasar/src/composables/github/types/index.ts

```diff
@@ -1,4 +1,2 @@
 //? This file simply acts as an aggregator for all types
-import Gist from './Gist';
-
-export { Gist };
+export { default as GistItem } from './Gist';
```

Comment:
Prefer named exports in types since that means we can have multiple related types in one file.
```ts
export * from './Gist';
```